internal/version/version_test.go

```diff
@@ -28,19 +28,16 @@ func TestTag(t *testing.T) {
 		t.Skip(err)
 	}
 	dates := strings.Split(string(bytes.TrimSpace(out)), "\n")
-	if len(dates) != 2 {
-		t.Skip("unexpected output: ", dates)
-	}
 	dateHEAD, err := unixDate(dates[0])
 	if err != nil {
 		t.Skip(err)
 	}
-	dateTag, err := unixDate(dates[1])
+	dateTag, err := unixDate(dates[len(dates)-1])
```

Comment:
This was needed because the test failed on tags generated with our tooling. It seems that a SSH signature is included in each tag, so the second date isn't at the end of a slice of two elements, but at the end of a longer slice.